packages/flutter/lib/src/widgets/text.dart

```diff
@@ -21,7 +21,7 @@ class DefaultTextStyle extends InheritedWidget {
     this.softWrap: true,
     this.overflow: TextOverflow.clip,
     this.maxLines,
-    Widget child
+    @required Widget child,
   }) : super(key: key, child: child) {
     assert(style != null);
     assert(softWrap != null);
```

Comment:
maybe mention in the `fallback` constructor that it'll create a version without a child, and thus one that can't be used in the tree itself?